app/models/apps/evolution_api.rb

```diff
@@ -37,6 +38,7 @@ class Apps::EvolutionApi < ApplicationRecord
     'active': 'active',
```

Comment:
Esse enum talvez se utilizarmos fica mais explícito o estado e não confunde com o campo  `active :boolean`
```suggestion
enum connection_status: {
    'disconnected': 'disconnected',
    'connected': 'connected',
    'sync': 'sync',
    'pair': 'pair',
}
```

Reply:
Boa! Substitui o pair pelo connecting tbm.